README.adoc

```diff
@@ -4950,6 +4950,28 @@ LOREM
 "when an unknown printer took a galley of type and scrambled it to make a type specimen book."
 ----
 
+== Enumerable
+
+=== lazy [[lazy]]
```

Comment:
I'd use a more descriptive heading here - something like `Leverage lazy to optimize memory usage` or whatever.